maths/elu_activation.py

```diff
@@ -0,0 +1,39 @@
+"""
+Implements the Exponential Linear Unit or ELU function.
+
+The function takes a vector of K real numbers and a real number alpha as
+input and then applies the ELU function to each element of the vector.
+
+Script inspired from its corresponding Wikipedia article
+https://en.wikipedia.org/wiki/Rectifier_(neural_networks)
+"""
+
+import numpy as np
+
+
+def elu_activation(vector: np.ndarray, alpha: float) -> np.ndarray:
```

Comment:
This repo resists using acronyms...
```suggestion
def exponential_linear_unit(vector: np.ndarray, alpha: float) -> np.ndarray:
```